spec/classes/backend_spec.rb

```diff
@@ -2,6 +2,9 @@
 
 describe 'sensu::backend', :type => :class do
   on_supported_os({facterversion: '3.8.0'}).each do |os, facts|
+    if facts[:os]['family'] == 'windows'
```

Comment:
add comment here that it's skipped because the backend is not supported on windows

Reply:
Comment added.